tests/src/com.hazelcast/hazelcast/5.2.1/src/test/resources/META-INF/native-image/javax.cache/cache-api/1.1.1/serialization-config.json

```diff
@@ -0,0 +1,18 @@
+{
+  "lambdaCapturingTypes": [],
+  "proxies": [],
+  "types": [
+    {
+      "name": "javax.cache.configuration.FactoryBuilder$SingletonFactory"
+    },
+    {
+      "name": "javax.cache.expiry.AccessedExpiryPolicy"
+    },
+    {
+      "name": "javax.cache.expiry.Duration"
+    },
+    {
+      "name": "javax.cache.expiry.EternalExpiryPolicy"
+    }
+  ]
+}
```

Comment:
- I think this is acceptable, and I cannot find these serializable json with Conditional Mode by changing `user-code-filter.json`. I had to manually filter `javax.cache:cache-api`'s GraalVM Reactablity Metadata using Standard Agent Mode.